lucene/core/src/java/org/apache/lucene/index/LiveIndexWriterConfig.java

```diff
@@ -416,7 +416,7 @@ public InfoStream getInfoStream() {
    * </p>
    */
   public LiveIndexWriterConfig setUseCompoundFile(boolean useCompoundFile) {
-    this.useCompoundFile = useCompoundFile;
+    this.useCompoundFile = false;
```

Comment:
same comment as below

Reply:
this thing also for remove compound file formation so we can see our file sizes more accurately